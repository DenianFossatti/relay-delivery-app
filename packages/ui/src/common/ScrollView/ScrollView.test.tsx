import { mockTestID, render, wrapper } from '../../';
import { ScrollView, ScrollViewProps, Text } from '../../components/atoms';

describe(`ScrollView`, () => {
  const props: ScrollViewProps = {
    children: <Text>Oi</Text>,
    testID: 'ScrollView',
  };

  const { findByTestId, toJSON } = render(<ScrollView {...props} />, {
    wrapper,
  });

  const el_container = mockTestID('View', props.testID!);

  it(`render component #${el_container}`, async () => {
    const sut = await findByTestId(el_container);
    expect(sut).toBeTruthy();
    expect(toJSON()).toMatchSnapshot();
  });
});
