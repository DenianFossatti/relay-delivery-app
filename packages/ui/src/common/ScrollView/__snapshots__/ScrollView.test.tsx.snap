// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/ScrollView render component #View_ScrollView 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RCTScrollView
    accessible={false}
    contentContainerStyle={
      [
        {},
        {
          "dataSet": {},
        },
        {},
      ]
    }
    dataSet={{}}
    style={{}}
    testID="View_ScrollView"
  >
    <View>
      <Text
        accessible={false}
        dataSet={{}}
        style={
          {
            "backgroundColor": undefined,
            "color": "#171717",
            "fontFamily": "Quicksand-Regular",
            "fontSize": 14,
            "fontStyle": undefined,
            "fontWeight": undefined,
            "letterSpacing": 0,
            "lineHeight": 21,
            "textDecorationLine": undefined,
          }
        }
        testID="Text_Text"
      >
        Oi
      </Text>
    </View>
  </RCTScrollView>
</RNCSafeAreaProvider>
`;
